CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+* Appium 1.13 will be the last minor version to support Node v8. As of Appium 1.14 the supported Node versions will be v10 and v12.
+
+#### Android
+* New capabilities:
+  * `remoteAppsCacheLimit`that sets the limit for how many APKs will be cached on a device [#523](https://github.com/appium/appium-android-driver/pull/523)
+  * `chromedriverPorts` which allows specifying multiple ports or a range of Chromedriver ports to use for web tests [#529](https://github.com/appium/appium-android-driver/pull/529)
+  * `buildToolsVersion` which allows you to set the Android `build-tools` version to be something different than the platform version [#532](https://github.com/appium/appium-android-driver/pull/532)
+* Fixes:
+  * Past version had a problem with emulators going offline when root/unroot was being called. Only affected unrooted emulators. Fix was to check if a device went offline after root/unroot was called and then restarting the ADB server [#443](https://github.com/appium/appium-adb/pull/443)
+  * Calls to `mobile:` endpoints weren't being called in web context. Defaults now is that in a web context, the native mobile endpoint is always called [#527](https://github.com/appium/appium-android-driver/pull/527)
+  * `pushFile` wasn't working on some later Android SDK's due to permission errors [#439](https://github.com/appium/appium-adb/pull/439)
+  * `network_connection` wasn't usable [#531](https://github.com/appium/appium-android-driver/pull/531)
+  * Default values in caps not being set correctly [#436](https://github.com/appium/appium-adb/pull/436)
+* No longer uninstalls apps when session is terminated if `dontStopAppOnReset` is set [#530](https://github.com/appium/appium-android-driver/pull/530)
+* Allow touch acitons in a web context [#534](https://github.com/appium/appium-android-driver/pull/534/files)
+
+#### Android (UiAutomator2)
+* Use UiAutomator's screenshot method when default screenshoter fails [#264](https://github.com/appium/appium-uiautomator2-server/pull/264)
+* Add default network to the device information endpoint (appium/device/info) [#265](https://github.com/appium/appium-uiautomator2-server/pull/265). Addresses [Issue #12502](https://github.com/appium/appium/issues/12502)
+
+#### iOS
+* Support webview testing for iOS real devices (1.12.1 already added support for iOS Simulators) [#122](https://github.com/appium/appium-remote-debugger/pull/122)
+* Supports tvOS [#151](https://github.com/appium/WebDriverAgent/pull/151)
```

Comment:
some more ios fixes/improvements have been done:

https://github.com/appium/appium/issues/12486
https://github.com/appium/appium/issues/12466
https://github.com/appium/appium/issues/12463

Reply:
- Can you link https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/ios/ios-tvos.md as the tvOS support? Users can go to the proper link to start tvOS test.
    - Can you address _updating your carthage to the latest one_ since carthage should handle simulator versions correctly for tvOS on simulator environment? Older carchage has no ability to handle them. (It can only build iOS platform) I'll add this in to `ios/ios-tvos.md` later tho.

And could you release newer https://github.com/appium/appium-xcuitest-driver/commits/master ? The latest commit (updating WDA) has a fix for tvOS support.